### ISSUE TITLE
STL: Update comments citing old Working Papers

### ISSUE THREAD

**Issue by StephanTLavavej** [microsoft/STL#182]
In comments, we often cite and quote the current Working Paper. Over the years, its sections are renumbered and its wording changes due to features being added, library issues being resolved, and editorial issues being cleaned up. This is especially true after a recent overhaul of Constraints/Mandates/Requires/etc. elements.

We should clean up these old comments, which involves:

1. Citing the now-current Working Paper
2. Updating the section number if it has changed, or dropping it if we decide that sections are renumbered too frequently
3. Checking that the stable name is still accurate (these rarely change, but it happens)
4. Updating the paragraph numbers if they have changed
5. Updating any quoted wording (making sure to clean out non-ASCII characters like long dashes)

**Comment by CaseyCarter** [comment]
All of our working draft citations will always be either out-of-date or at most a few months away from being out-of-date: the working drafts are a clock that never stops ticking. Given that we can't stay current without someone committing to updating every citation every time WG21 publishes a new working draft - which seems like effort that could be better invested elsewhere - I assume your intent is that we update all citations to working drafts older than X? Or perhaps that we update all citations to the final pre-publication working draft every time WG21 publishes a standard? (Once every three years seems more reasonable than once every four months.)

**Comment by StephanTLavavej** [comment]
Yeah, I think updating to WG21-N4835 now, and then to each Standard, is a reasonable concrete goal.

**Comment by BillyONeal** [comment]
WG21-N4849 is the current version as of this comment.
